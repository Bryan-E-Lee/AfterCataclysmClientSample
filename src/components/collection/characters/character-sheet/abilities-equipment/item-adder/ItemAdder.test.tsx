// import { render, screen } from '@testing-library/react';
// import renderer from 'react-test-renderer';
// import 'jest';
// import React from 'react';
// import { ItemAdder } from '.';
// import { Character } from '../../../../../../entities/characters/Character';
// import { SheetDefaultState } from '../../../../../../store/stores/characters/sheet/CharacterSheet.State';
// import { SortedSet } from '../../../../../../entities/data-structures/SortedSet';
// import { ItemInitializer } from '../../../../../../entities/library/items/ItemInitializers';
// import { RhetoricInitializer } from '../../../../../../entities/library/socials/Rhetoric';
// import { MinionInitializer } from '../../../../../../entities/library/minions/Minion';
// import { PerkInitializer } from '../../../../../../entities/library/perks/Perk';
// import { SkillInitializer } from '../../../../../../entities/library/skills/Skill';
// import { PersonalityInitializer } from '../../../../../../entities/library/socials/Personality';

// const createDefaultCharacter = (): Character => {
//     return new Character(SheetDefaultState, {
//         items: new SortedSet<ItemInitializer>(),
//         minions: new SortedSet<MinionInitializer>(),
//         perks: new SortedSet<PerkInitializer>(),
//         personalities: new SortedSet<PersonalityInitializer>(),
//         rhetorics: new SortedSet<RhetoricInitializer>(),
//         skills: new SortedSet<SkillInitializer>(),
//     });
// }

// describe('item adder', () => {
//     describe('render', () => {
//         test('default state snapshot', () => {
//             const adder = <ItemAdder character={createDefaultCharacter()} close={() => {}} />;
//             const component = renderer.create(adder);

//             const tree = component.toJSON();

//             expect(tree).toMatchSnapshot();
//         })

//         test('add to container state snapshot', () => {
//             const adder = <ItemAdder character={createDefaultCharacter()} close={() => {}} />;
//             const component = renderer.create(adder);

//             const tree = component.toJSON();

//             expect(tree).toMatchSnapshot();
//         })
//     })
// })