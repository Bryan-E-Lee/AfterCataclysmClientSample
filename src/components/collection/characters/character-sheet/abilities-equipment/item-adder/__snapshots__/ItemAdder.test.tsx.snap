// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`item adder render add to container state snapshot 1`] = `
<div
  className="overlay inactive"
/>
`;

exports[`item adder render default state snapshot 1`] = `
<div
  className="overlay inactive"
/>
`;
